hammer/library/aws/redshift.py

```diff
@@ -0,0 +1,259 @@
+    'id',
+    # subnet_group_id
+    'subnet_group_name'
+    ])
+
+
+class RedshiftClusterOperations(object):
+
+    @classmethod
+    @timeit
+    def get_redshift_vpc_security_groups(cls, redshift_client, group_id):
+        """ Retrieve redshift clusters meta data with security group attached
+
+                :param redshift_client: boto3 redshift client
+                :param group_id: security group id
+
+                :return: list with redshift clusters details
+                """
+        # describe rds instances with security group attached
+        redshift_clusters = []
+
+        # this will include Clusters
+        clusters_res = redshift_client.describe_clusters()
+        for cluster in clusters_res["Clusters"]:
+            active_security_groups = [sg["VpcSecurityGroupId"] for sg in cluster['VpcSecurityGroups'] if
+                                      sg["Status"] == "active"]
+            if group_id in active_security_groups:
+                redshift_clusters.append(RedshiftCluster_Details(
+                    id=cluster["ClusterIdentifier"],
+                    subnet_group_name=cluster["ClusterSubnetGroupName"]
+                ))
+
+        return redshift_clusters
+
+    @staticmethod
+    def set_cluster_access(redshift_client, cluster_id, public_access):
+        """
+        Sets the cluster access as private.
+
+        :param redshift_client: Redshift boto3 client
+        :param cluster_id: Redshift cluster name which to make as private
+        :param public_access: Redshift cluster public access True or False.
+
+        :return: nothing
+        """
+
+        redshift_client.modify_cluster(
+            ClusterIdentifier=cluster_id,
+            PubliclyAccessible=public_access
+        )
+
+    @staticmethod
+    def cluster_encryption(redshift_client, cluster_id):
+        """
+        
+        :param redshift_client: redshift client
+        :param cluster_id: cluster id which need to be encrypted. 
+        
+        :return: 
+        """
+        # Modify cluster as encrypted.
+        redshift_client.modify_cluster(
+            ClusterIdentifier=cluster_id,
+            Encrypted=True
+        )
+
+
+class RedshiftCluster(object):
+    """
+    Basic class for Redshift Cluster.
+    Encapsulates `Owner`/`Tags`.
+    """
+    def __init__(self, account, name, tags, is_encrypted=None, is_public=None, is_logging=None):
+        """
+        :param account: `Account` instance where redshift cluster is present
+
+        :param name: `Name` of cluster id
+        :param tags: tags if redshift cluster tags (as AWS returns)
+        :param is_encrypted: encrypted or not.
+        """
+        self.account = account
+        self.name = name
+        self.tags = convert_tags(tags)
+        self.is_encrypt = is_encrypted
+        self.is_public = is_public
+        self.is_logging = is_logging
+
+    def modify_cluster(self, public_access):
+        """
+        Modify cluster as private.
```

Comment:
I think we pass only `False` value, so argument is not needed.
We may also name this method as `make_private` for better understanding of what it does.